frontend/src/components/containers/README.md

```diff
@@ -0,0 +1,5 @@
+### Containers
```

Comment:
Is calling your logical components `containers` a convention? Inside of the `components` directory, why not have to directories called `logical` and `presentational`?

Reply:
Mainly just asking the question, not requesting a change here

Reply:
"containers" is the name by convention. It's a bit less specific than "logical", which might tend to make it a dumping ground for any component with logic.  They are logical components, but their main distinguishing feature is that they do not render display-level JSX, they render other components. Essentially, a container is something that "Contains" presentational components. Maybe sounds like splitting hairs, but I think I can see why the convention is to call them "Containers".